src/main/scala/org/allenai/plugins/CoreDependencies.scala

```diff
@@ -79,7 +79,7 @@ trait CoreDependencies {
   val sprayCaching = sprayModule("caching")
 
   // Spray json (separate from Spray toolkit)
-  val sprayJson = "io.spray" %% "spray-json" % "1.2.6"
+  val sprayJson = "io.spray" %% "spray-json" % "1.3.1"
```

Comment:
Also, since when are API-breaking changes allowed in minor version changes? *shakes fist*